src/calendar/calendarSection/CalendarView.js

```diff
@@ -16,8 +16,10 @@ function CalendarView() {
 
   return (
     <div className="flex column">
-      <Weekdays />
-      <CalendarDays />
+      <div id="calendarBox">
+        <Weekdays />
```

Comment:
You add additional markup to align the calendar box at the start. Wouldn't it be better to add alignItems:start onto the _.wrapper_ ?

Reply:
PS - Ignore the above- if doing it,  the Event Box will be part of the Calendar. Having the div solves more problems : )